core/modes/sleep_mode_pop_twice_to_wake.py

```diff
@@ -0,0 +1,45 @@
+import time
+
+from talon import Context, Module, actions, settings
+
+ctx = Context()
+mod = Module()
+
+mod.tag("pop_twice_to_wake", desc="tag for enabling pop twice to wake in sleep mode")
+
+mod.setting(
+    "last_pop_time_maximum",
```

Comment:
I think I'd lean towards something like

```suggestion
    "double_pop_speed_minimum",
```